### DISCUSSION TITLE
Catppuccin Keycaps

### DISCUSSION THREAD

**Discussion by theodotdot** [catppuccin/catppuccin#2407] [Ideas]
This is an idea that has been thrown around a bit in the Discord server but something that I feel would be very successful and I would like to try to move that project forward. However there are major roadblocks to go over (at least those that I can think of for now):

## 1. Creating a proper interest check on Geekhack
This is **a** part of showing off the project, gauging interest and finding potential vendors and manufacturers.
For this we would need a solid design proposal, and a bunch of high quality renders to show those designs off.
One thing that is great is that, in a normal project, finding the colours is a major part but since Catppuccin has already predefined colours and guidelines on how to use those colours that simplifies the work.

## 2. Finding and coordinating with a manufacturer and with vendors
I have absolutely no experience in this and from quick discussions with people, catppuccin's colours don't exactly translate well in the physical world (i.e. colour matching will be hell and could lead to disappointing results if not done well)

## 3. All the money talk
This will require some money up front. That is something that I can potentially help with. I don't plan to make money off of this but rather to cover any costs occured and I think it would be fair that profits would go to the Open Collective Catppuccin fund.

I am trying to reach out to people I know in the keyboard community to see if I can find some help and/or guidance and will update this post accordingly. But if you are someone willing to offer help or anything that you see would be interesting, feel free to jump in!

Thank you!

EDIT: here are some quick renders of what could be possible using the catppuccin colours:
![image](https://github.com/catppuccin/catppuccin/assets/59372255/ee87e9a7-35ac-4d0e-b502-9549f09579e1)
![image](https://github.com/catppuccin/catppuccin/assets/59372255/c7a6242a-d225-4860-b1c7-58f4a24d3c53)

**Comment by SomeGuyNamedMay** [comment]
i would personally love to see a catppuccin themed set for a heavily sculpted profile like mt3 or sa

**Comment by shadow-absorber** [comment]
would absolutely love to have this as an option

**Comment by pernydev** [comment]
This would be SO AWESOME!

**Comment by shadow-absorber** [comment]
considering [https://github.com/catppuccin/pantone](https://github.com/catppuccin/pantone) exists there is some progress towards what the keycaps should look like in the pantone color system

**Comment by jaashuu** [comment]
if anyone absolutely desperately needed something close, there is the KAT milkshake dark, which i think definitely reflects the aesthetic of catppuccin dark for sure

**Comment by pernydev** [comment]
I already mentioned this in the Discord server. But I have a friend who has done large-scale keycap manufacturing (and all the logistics behind that) and we'd love to work on this! I think what would be the best way to go on about this is a crowdfunding-style pre-order campaign.

Oh, and here's a mockup of what we're currently working with! (it's still early)
![Frame 1(12)](https://github.com/user-attachments/assets/6c651878-2e74-44bd-a261-53291c908b92)

**Reply by shadow-absorber** [reply]
are you using the pantone colors for this or how are you getting the colors???

**Reply by pernydev** [reply]
Our supplier said that they will be able to work with CMYK, the pantones are too off in my opinion.

**Reply by shadow-absorber** [reply]
nice. as long as the keycaps looks good shadow will be happy

**Reply by shadow-absorber** [reply]
also as a swedish person and citizen would love iso compability

**Reply by pernydev** [reply]
As a Finnish person, yes we need ISO.

**Comment by leavism** [comment]
I'm on YUZU trying to get a Catppuccin Latte ish color scheme going for my silver aluminum corne. YUZU has a set of color palettes that they work with so I'm limited to those, but they let you upload custom fonts and icons.

Anyone have any recommendation on font or icons I could use? Or any improvements in general to what I have? Here's a link if anyone would like to mess around:
https://yuzukeycaps.com/c/cd7a75fd-3bf9-42f7-b702-30d1fcb908df

The letter keycaps look kinda creamy but apparently they come out as white. I'll have to clarify with YUZU when I submit the design. 
![yuzu](https://github.com/user-attachments/assets/2a1e3f6f-35ee-4b32-9c3e-acbcb384aacd)

**Reply by Furglitch** [reply]
Personally I'm a fan of the Ubuntu font from NerdFonts. Feels kinda like a modern classic.

**Comment by MinweiShen** [comment]
We designed this and is considering manufacturing it. Love to hear some suggestions!
<img width="1200" height="675" alt="catppuccin keycaps" src="https://github.com/user-attachments/assets/b5f11343-316b-496b-be6d-58c5b6f331d5" />

**Reply by shadow-absorber** [reply]
looks good but would love for there to be a full size keyboard keycap kit

**Comment by MinweiShen** [comment]
For those who might be interested: after months of work, we made it! My keyboard can now match my editors.

<img width="990" height="660" alt="image" src="https://github.com/user-attachments/assets/dc4a79ec-c130-4312-b62a-5eb253f791c8" />
<img width="900" height="900" alt="image" src="https://github.com/user-attachments/assets/faaf8770-c9e8-4a87-a930-a72002f79227" />
<img width="900" height="1203" alt="image" src="https://github.com/user-attachments/assets/0f1e9c23-c42e-4e4c-9dbb-4acf6829d68a" />


If you are also interested, here's a promo code with 15% off (once added to cart): 
[Catppuccin theme pbt keycaps from PetitKeycaps](https://petitkeycaps.com/discount/FV3ZG84RZVA2?redirect=%2Fproducts%2Fcatppuccin-theme-pbt-keycaps%3Futm_campaign%3Dgithub)

**Reply by uncenter** [reply]
I wish the contrast was better for the text/labels. White on the base keys and base/crust on the accent keys would be so nice!

**Reply by MinweiShen** [reply]
> I wish the contrast was better for the text/labels. White on the base keys and base/crust on the accent keys would be so nice!

Got it! We will take into account in our V2 version :)

**Reply by uncenter** [reply]
This design has what I was looking for fwiw:
https://github.com/catppuccin/catppuccin/discussions/2407#discussioncomment-12781464

**Reply by coryshaw1** [reply]
> This design has what I was looking for fwiw: [#2407 (comment)](https://github.com/catppuccin/catppuccin/discussions/2407#discussioncomment-12781464)

Completely agree. The contrast of the production version isn't enough. The linked design is better IMO.

**Reply by shadow-absorber** [reply]
ordered these.... better then nothing but agree with the previous comments... also for european purchasers you need to add a phone number to be able to get these keycaps shipped to you

**Comment by MinweiShen** [comment]
Hi all,
Want to share some updates after hearing your feedback: we made a V2 version with improved contrast and a new set of novelty keys. 

Here's the link:
- base kit: https://petitkeycaps.com/products/catppuccin-theme-pbt-keycaps
- novelty set: https://petitkeycaps.com/products/catppuccin-paws-r1-cherry-profile-arrow-key-set

Let me know your thoughts!

Some pictures:

<img width="1920" height="1920" alt="image" src="https://github.com/user-attachments/assets/aaedf1b2-b2c6-47fa-8980-f2da14112a34" />
<img width="1920" height="1920" alt="image" src="https://github.com/user-attachments/assets/6695e629-552e-47b6-b597-974ea8846534" />
<img width="1920" height="1280" alt="image" src="https://github.com/user-attachments/assets/a51bcae9-465e-4dba-96ff-0f10ef0bfffc" />

**Comment by Neraud** [comment]
The set from MinweiShen is gorgeous, but it's obviously only available in English.

Has anyone tried to create Catppuccin color schemes on sites like Yuzu (like [leavism started for latte](https://github.com/catppuccin/catppuccin/discussions/2407#discussioncomment-13477298)) ?

**Comment by MinweiShen** [comment]
Hi everyone, had been working on a light version and now it's live! 
https://petitkeycaps.com/products/catppuccin-keycap-light

The discussion is where the idea of making a keyboard set originated from and I did get some feedback here. This meant a lot for me and I really appreciate it. Love to hear your thoughts on the new set.

<img width="1920" height="1920" alt="DSC00460 拷贝" src="https://github.com/user-attachments/assets/b9f2fe9a-2626-44ba-bc68-15c77f264516" />
<img width="1920" height="1536" alt="cat light 3" src="https://github.com/user-attachments/assets/07dc3e02-61ae-465d-adb6-e14a921a1554" />
<img width="1920" height="1280" alt="cat light 2" src="https://github.com/user-attachments/assets/5174b4f4-6940-443a-93af-becbffd8c432" />

**Reply by SherLock707** [reply]
Looks awesome! Can we have XDA profile please?

**Reply by MinweiShen** [reply]
> Looks awesome! Can we have XDA profile please?

Not planned right now unfortunately. Hope one day we can expand to other profiles and layouts

**Reply by coryshaw1** [reply]
This turned out pretty good! +1 to XDA, DSA, etc. Any uniform layout would be an instant buy especially for non-standard ergo layouts
